website/docs/r/pi_volume_clone.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_volume_clone"
+description: |-
+   Manages IBM Volume Clone in the Power Virtual Server cloud.
+---
+
+# ibm_pi_volume_clone
+Create a volume clone. For more information, about managing volume clone, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
+
+## Example usage
+The following example creates a volume clone.
+
+```terraform
+resource "ibm_pi_volume_clone" "testacc_volume_clone" {
+  pi_cloud_instance_id   		= "<value of the cloud_instance_id>"
+  pi_volume_clone_name     	    = "test-volume-clone"
+  pi_volume_ids                 = ["<Volume ID>"]
+  pi_target_storage_tier        = "<storage tier>"
+  pi_replication_enabled        = true
+}
+```
+
+**Note**
+* Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+* If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  * `region` - `lon`
+  * `zone` - `lon04`
+
+  Example usage:
+  
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+  
+## Timeouts
+
+ibm_pi_volume_clone provides the following [timeouts](https://www.terraform.io/docs/language/resources/syntax.html) configuration options:
+
+- **create** - (Default 15 minutes) Used for creating volume clone.
+- **delete** - (Default 15 minutes) Used for deleting volume clone.
+
+## Argument reference 
+Review the argument references that you can specify for your resource. 
+
+- `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
+- `pi_replication_enabled` - (Optional, Boolean) Indicates whether the cloned volume should have replication enabled. If no value is provided, it will default to the replication status of the source volume(s).
+- `pi_target_storage_tier` - (Optional, String) The storage tier for the cloned volume(s).
+- `pi_volume_clone_name` - (Required, String) The base name of the newly cloned volume(s).
+- `pi_volume_ids` - (Required, Set of String) List of volumes to be cloned.
+
+## Attribute reference
+In addition to all argument reference list, you can access the following attribute reference after your resource is created.
+
+- `cloned_volumes` - (List of objects) The List of cloned volumes.
+  Nested scheme for `cloned_volumes`:
```

Comment:
move this down.

Reply:
Should it be moved to the end? Its current position at the top is based on alphabetical order

Reply:
Got it. Document formatting has been fixed.